packages/atomic/src/components/common/tab-manager/tab-button.pcss

```diff
@@ -0,0 +1,21 @@
+:host {
+  border-radius: 60px;
```

Comment:
Most of those could be set with tailwind classes, no ? 
Or am I mistaken ?

Also those arbitrary variable (height/radius) should be controlled by --atomic variables

Reply:
Yep, It now uses tailwind classes. Also instead setting atomic variables for the tab bar thickness/position, I use normal tailwind classes, and have added a css part to the button container so that it is easy to style the bar.